### PR TITLE
Ep dspdc 1663 google beta provider

### PR DESCRIPTION
**Pull request by eparkmonster**
## Why
Want to swap out our Terraform config to utilize google instead of google-beta provider
[Relevant ticket](https://broadinstitute.atlassian.net/browse/DSPDC-1663)

## This PR
Swaps google-beta -> google provider and was tested locally by performing terraform plan after each change in a file

### CODE REVIEW

File: environments/hca-prod/terraform/provider.tf [hunk 1]
```diff
@@ -4,12 +4,18 @@ provider google-beta {
   project = local.prod_project_id
   region  = "us-central1"
 }
-provider google-beta {
+provider google{
```
Comment (aherbst-broad) [inline environments/hca-prod/terraform/provider.tf:7]: This is likely what `terraform fmt` is choking on.

**Review by aherbst-broad — COMMENTED** [review]
This looks good, I think you just need to run `terraform fmt --recursive` from the `environments/` directory and push those changes to get checks passing. Once things are green I'll approve.

File: environments/hca-prod/terraform/provider.tf [hunk 2]
```diff
@@ -4,12 +4,18 @@ provider google-beta {
...
   alias   = "prod-core"
   project = "broad-dsp-monster-prod"
   region  = "us-central1"
 }
 
+provider google {
```
Comment (ThePletch) [inline environments/hca-prod/terraform/provider.tf:13]: shouldn't this be replacing the provider on line 1, rather than being added as a new provider?

**Review by ThePletch — APPROVED** [review]
hoping this can be as simple as a find-replace, but we definitely want to see a plan come up clean before shipping this. just gotta fix that formatting error.

we should probably add a ticket to make the CI not choke on a failed `terraform fmt`.

**Review by aherbst-broad — APPROVED** [review]
I took a look at the plans; the dap changes are due to me not adding the related envs to the merge workflow (remedied in this PR https://github.com/broadinstitute/monster-deploy/pull/142). The HCA changes are a bit mystifying, I'm not sure why they drifted out of whack (possibly due to someone like myself running terraform apply locally). Either way, those changes should be ok.

**Pull request merged**